docs/commands/hexdump.md

```diff
@@ -2,10 +2,12 @@
 
 Imitation of the WinDBG command.
 
-This command requires at least 2 arguments, the format for representing the
-data, and a value/address/symbol used as the location to print the hexdump
-from. An optional 3rd argument is used to specify the number of
-qword/dword/word/bytes to display.
+This command takes 4 optional arguments:
+
+  - The format for representing the data (by default, byte)
+  - A value/address/symbol used as the location to print the hexdump from (by default, $sp)
+  - The number of qword/dword/word/bytes to display (by default, 64 if the format is byte, 16 otherwise)
+  - The directionof output lines (by default, from low to high addresses)
```

Comment:
`directionof `

Reply:
I'm glad that you liked the patch. I have just fixed the typo with the latest commit.